docs/dev_guide/maintainer_workflow.rst

```diff
@@ -0,0 +1,143 @@
+
+This page is for maintainers |emdash| those of us who merge our own or other
+peoples' changes into the upstream repository.
+
+Being as how you're a maintainer, you are completely on top of the basic stuff
+in :ref:`version_control`.
+
+Integrating changes
+-------------------
+
+Whenever possible, merge pull requests automatically via the pull request manager on GitHub. Merging should only be done manually if there is a really good reason to do this!
+
+Please try to make sure that pull requests do not contain a messy history with merges, etc. If this is the case, then you have two options. To use the rebase merge on Github or to follow the manual instructions, and make sure the fork is rebased to tidy the history before committing. The second option should only be used on complex pull requests.
+
+
+Using Milestones and Labels
+===========================
+
+These guidelines are adapted from `similar guidelines <http://docs.astropy.org/en/stable/development/workflow/maintainer_workflow.html#using-milestones-and-labels>`_
+.. _milestones-and-labels:
+followed by AstroPy:
+
+* 100% of confirmed issues and new features should have a milestone
+
+* Only the following criteria should result in an issue being closed without a milestone:
+
+  * Not actually an issue (user error, etc.)
+
+  * Duplicate of an existing issue
+
+  * A pull request superseded by a new pull request providing an alternate implementation
+
+* Open issues should only lack a milestone if:
+
+  * More clarification is required
+
+  * Which milestone it belongs in requires some discussion
+
+* Corollary: When an issue is closed without a milestone that means that the issue will not be fixed, or that it was
+  not a real issue at all.
+
+* In general there should be the following open milestones:
+
+  * The next bug fix releases for any still-supported version lines; for example if 0.4 is in development and
+    0.2.x and 0.3.x are still supported there should be milestones for the next 0.2.x and 0.3.x releases.
+
+  * The next X.Y release, i.e. the next minor release; this is generally the next release that all development in
+    master is aimed toward.
+
+  * The next X.Y release +1; for example if 0.3 is the next release, there should also be a milestone for 0.4 for
+    issues that are important, but that we know won't be resolved in the next release.
+
+  * Future; this is for all issues that require attention at some point but for which no immediate solution is in
+    sight.
+
+* Bug fix release milestones should only be used for deferring issues that won't be fixed in the next minor release,
+  or for issues is previous releases that no longer apply to the mainline.
+
+* When in doubt about which milestone to use for an issue, use the next minor release, it can always be moved once
+  it's been more closely reviewed prior to release.
+
+* Active milestones associated with a specific release (eg. v0.3.0) should contain at least one issue with the
```

Comment:
we don't do this any more, but rather has a very detailed release prodecure document: http://docs.astropy.org/en/latest/development/releasing.html

Reply:
Thats very nice, I might have to create one for SunPy as well